src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,46 @@
+.bp-ColorPickerPalette {
+    position: absolute;
+    left: -55px;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 35px;
+    margin-bottom: 125px;
+    background: inherit;
+    border-radius: 5px;
```

Comment:
Can we keep the border radii consistent with the other controls? Otherwise it will look out of place.

Reply:
Yep, will fix.